### DISCUSSION TITLE
Do I need remark to use plugins?

### DISCUSSION THREAD

**Discussion by louis-vinchon** [micromark/micromark#74] [Q&A] [answered]
I wanted to have rendered links open in a new tab, and the documentation nicely pointed me to [remark-external-links](https://github.com/remarkjs/remark-external-links).

However, neither documentations mention how it's supposed to be used with micromark.

Do I need to upgrade to remark in order to use plugins?

**Comment by wooorm** [comment, marked as answer]
Yes
